README.md

```diff
@@ -93,16 +111,20 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
-
+```
+You call a function that you want executed now by writing it as someVariable = someFunction (). You reference a function that you want to run upon some other action preceding it by writing it as someVariable = someFunction. This would imply that someVariable is an alias for someFunction. If we call someVariable(), we would get the same result as someFunction().
 ```
 ### Question #9
 
 Using the object literal notation, Define an object called student and give it the properties (your attributes) of name, age, and a method sayHello, that outputs "Hi, my name is [your_name]".
 
 Your Answer:
-```js
-// write code here
+```
+var student = {
+  name: "Robel",
+  age: "37"
+}
+console.log("Hi, my name is " + student.name);
 ```
```

Comment:
Close - would just rethink what the question is asking here. The prompt is define a method, a function as a property on an object, that utilizes other properties of that object in the function definition. i.e.

```js
 var student = {
   name: "Robel",
   age: 37.
   sayHello: function( ) {
      // inside this method definition we can access other properties of the object
      console.log("Hi, my name is " + this.name)
  }
}

student.sayHello( )
=> "Hi, my name is Robel"
```